pipelines/skylab/snm3C/snm3C.wdl

```diff
@@ -23,20 +26,32 @@ workflow snm3C {
         Int num_downstr_bases = 2
         Int compress_level = 5
         Int batch_number
-        String docker = "us.gcr.io/broad-gotc-prod/m3c-yap-hisat:2.4"
+    }
+    # Determine docker prefix based on cloud provider
+    String gcr_docker = "us.gcr.io/broad-gotc-prod/m3c-yap-hisat:2.4"
```

Comment:
```suggestion
    String gcr_docker_prefix = "us.gcr.io/broad-gotc-prod/"
```
can we break it down like this? just to keep consistency with how we have azurized other wdls